### PR TITLE
[master] - Development

### PR DESCRIPTION
**Pull request by leonkozlowski**
Very basic - implemented one `Flow`

Using a base class `EventBrokerFlow`
we will implement individual flows for bet365

`UpcomingEventsFlow`
Tasks:
-GET Request to `Upcoming Events` endpoint via`betfund-bet365`
-PUT (send) to Kafka topic "upcomingEvents"

### CODE REVIEW

File: betfund_event_broker/flows/base.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        Returns:
+            State: State of reference tasks in Prefect Flow
+        """
+        flow = self.build(*args, **kwargs)
+        flow_state = self.execute(
+            flow=flow, *args, **kwargs
+        )
+
+        return flow_state.serialize()
```
Comment (mitchbregs) [inline betfund_event_broker/flows/base.py:90]: This class right here fires me up

File: betfund_event_broker/tasks/bet365/prematch_odds.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            (e.g.)
+                "87941408"
+
+        Returns:
+            tuple: contains API response object and kafka topic
+        """
+        bet365_client = self._build_client()
+
+        if not fi:
+            return None, self.topic  # this needs to be amended
```
Comment (mitchbregs) [inline betfund_event_broker/tasks/bet365/prematch_odds.py:37]: What happens here?

**Review by mitchbregs — APPROVED** [review]
🤑 PR - love this.  If this is going to `master`, can you through up a README with `Installation, Usage, Tests`?  Good to merge.

**Comment by leonkozlowski** [comment]
> 🤑 PR - love this. If this is going to `master`, can you through up a README with `Installation, Usage, Tests`? Good to merge.

Sure thing

File: betfund_event_broker/tasks/bet365/prematch_odds.py (hunk 2 above)
Comment (leonkozlowski) [reply betfund_event_broker/tasks/bet365/prematch_odds.py:37]: Still need to look into graceful error handling for this

**Pull request merged**